tests/models/luke/test_modeling_luke.py

```diff
@@ -139,15 +147,20 @@ def prepare_config_and_inputs(self):
             )
 
         sequence_labels = None
-        labels = None
+        token_labels = None
+        choice_labels = None
+        # labels = None
```

Comment:
To be removed?

Reply:
This line is removed in the latest commit